factory/usr/lib/the-modeenv

```diff
@@ -19,6 +19,13 @@ if [ "${mode}" = "run" ]; then
     elif [ -f /run/mnt/ubuntu-boot/uboot/ubuntu/boot.sel ]; then
         echo '/run/mnt/ubuntu-boot/uboot/ubuntu /boot/uboot none bind 0 0' >> /run/image.fstab
     fi
+    # if there is override for journald.conf, remove it, it has ForwardToConsole=true
+    # also modify generator, as current mode is now new run mode
+    if [ -f /run/systemd/journald.conf.d/core-override.conf ]; then
+        rm /run/systemd/journald.conf.d/core-override.conf
+        cmdline_mode="$(/usr/libexec/core/get-arg snapd_recovery_mode)
+        sed -i 's/snapd_recovery_mode=run/snapd_recovery_mode='"${cmdline_mode}"'/g' /usr/lib/systemd/system-generators/journald-console
+    fi
```

Comment:
is this hunk still needed, given the improved generator? cause in core22 pull request it is now not needed, or am i missing some other core20 vs core22 differences?